OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -100,7 +100,7 @@ public override void Draw()
 			var bgDark = GetContrastColorDark();
 			var bgLight = GetContrastColorLight();
 			if (Contrast)
-				font.DrawTextWithContrast(text, position, color, bgDark, bgLight, 2);
+				font.DrawTextWithContrast(text, position, color, bgDark, bgLight, 1);
```

Comment:
I feel like this should be configurable in UI metrics rather than being hard-coded here.

Reply:
Done.